src/features/firstLogin/PrivacyPolicy/PrivacyPolicy.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { act, fireEvent, render } from '@testing-library/react-native'
+import React from 'react'
+import waitForExpect from 'wait-for-expect'
+
+import { storage } from 'libs/storage'
+import { flushAllPromises } from 'tests/utils'
+
+import { PrivacyPolicy } from './PrivacyPolicy'
+
+describe('PrivacyPolicy page', () => {
+  afterEach(() => {
+    storage.clear('has_accepted_cookie')
+  })
+  it('should render privacy policy', async () => {
+    jest.useFakeTimers()
+    const firstTutorial = render(<PrivacyPolicy />)
+    await act(flushAllPromises)
+    jest.advanceTimersByTime(1000)
+    expect(firstTutorial).toMatchSnapshot()
+    jest.useRealTimers()
+  })
+
+  it('should show modal when has_accepted_cookie is undefined', async () => {
+    await storage.saveObject('has_accepted_cookie', null)
+    const renderAPI = render(<PrivacyPolicy />)
+    await act(flushAllPromises)
+    await waitForExpect(() => {
+      const title = renderAPI.queryByText('Respect de ta vie privée')
+      expect(title).toBeTruthy()
+    })
```

Comment:
La plupart du temps, si tu flush les promises, tu n'as pas besoin de wait par la suite : 
```suggestion
      const title = renderAPI.queryByText('Respect de ta vie privée')
      expect(title).toBeTruthy()
```